jena/src/main/java/org/apache/commons/rdf/jena/JenaRDFTerm.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.commons.rdf.jena;
+
+import org.apache.commons.rdf.api.RDFTerm;
+import org.apache.jena.graph.Node;
+
+/** Access the Jena node backing this object */
+public interface JenaRDFTerm extends RDFTerm {
```

Comment:
Another problem with typing this I see now are some of the more special Node types in Jena that confuses me:

`Node_Blank`, `Node_NULL` etc.  


We don't use the specific Node interfaces within this integration, as `Node` itself has the required `.isBlank()` etc methods - I don't want to second-guess if `.isBlank()` == true means it has to be a `Node_Blank` instance; this sounds more like an internal interface.